src/containers/Posts/FullPost/FullPost.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import './FullPost.css';
+import { connect } from 'react-redux';
+import * as actions from '../../../store/actions/index';
+import Modal from '../../../components/UI/Modal/Modal';
+import UpdatePost from '../updatePost/updatePost';
+class FullPost extends Component {
+     state ={
+        updateModal : false
+    }  
+
+    componentDidMount() {
+        this.props.getPostById(this.props.token, this.props.userId, this.props.match.params.id);
+    }
+
+    deletePostHandler = () => {
+        this.props.deletePost(this.props.token, this.props.userId, this.props.match.params.id);
+    }
+
+    updatePostHandler = () => {
+        this.setState({updateModal : true})
+        this.props.updatePost(this.props.token, this.props.userId, this.props.match.params.id, { 'title': 'title' });
+    }
+    
+    closeModalHandler = () => {
+        this.setState({updateModal : false})
+    }
+
+    render() {
+        let posts = this.props.posts.map(post => {
+            return (
+                <div className="FullPost">
```

Comment:
If not post available then show message.

Reply:
I tried but it was showing message after logout also. Need fix it.

Reply:
Please fix the issue